src/controllers/user.js

```diff
@@ -57,11 +57,30 @@ export const getAll = async (req, res) => {
 }
 
 export const updateById = async (req, res) => {
-  const { cohort_id: cohortId } = req.body
-
-  if (!cohortId) {
-    return sendDataResponse(res, 400, { cohort_id: 'Cohort ID is required' })
+  const id = parseInt(req.params.id)
```

Comment:
maybe try catch this, if so return response and update openapi.yml accordingly

Reply:
The pre-existing solution for getById does not bother with this validation, for what it is worth.

Reply:
its nice to have, but not necessary

Reply:
but yeah, as you said, if added, implement in middleware